tuf/api/metadata.py

```diff
@@ -697,6 +697,14 @@ class Timestamp(Signed):
 
     _signed_type = "timestamp"
 
+    @property
+    def snapshot_meta(self):
+        return self.meta["snapshot.json"]
+
+    @snapshot_meta.setter
+    def snapshot_meta(self, value):
```

Comment:
missing annotation

(we should start tightening mypy config so it complains about this)